src/Bonus.java

```diff
@@ -1,23 +1,26 @@
 import java.util.HashSet;
 import java.util.Objects;
+import java.util.Random;
 import java.util.Scanner;
 
 public class Bonus {
 
+
+
     public static void main(String[] args) {
 
         // Onderstaande code is uitgecommentarieerd. Haal de code uit de comment (delete /* en */) om aan de slag te gaan met de bonus
         // In de code staan op verschillende plekken comments tussen "/*...*/" als hint wat je moet doen
 
 
         HashSet<Integer> secretnumber = randomnumbergenerator();
-        String stringnumber = setToStringConverter(secretnumber);
+        StringBuilder stringnumber = setToStringConverter(secretnumber);
         System.out.println(stringnumber);
-        feedback(/*vul hier het juiste argument in*/);
+        feedback (String.valueOf(stringnumber));
 
     }
 
-    public static void/*moet dit returntype "void" zijn of wat anders?*/ randomnumbergenerator(/*Heeft deze methode nog parameter(s) nodig?*/){
+    public static HashSet<Integer> randomnumbergenerator(/*Heeft deze methode nog parameter(s) nodig?*/ ){
```

Comment:
Deze methode heeft geen extra parameter nodig. Gezien je zelf een random variabele in de body instantieert.